xgettext.js

```diff
@@ -15,6 +15,7 @@ var XGettext = module.exports = function( options ) {
 	this.options = _.extend( {}, XGettext.defaultOptions, options );
 	this.options.keywords = this._normalizeKeywords( this.options.keywords );
 	this.options.keywordFunctions = Object.keys( this.options.keywords );
+	this.options.parserOptions = _.extend( {}, options.parserOptions );
```

Comment:
Do we need to create a copy of `parseOptions` ? This should already be injected into `this.options` above on line 15.